### DISCUSSION TITLE
How to change auth type?

### DISCUSSION THREAD

**Discussion by moogamouth** [internetarchive/heritrix3#588] [Q&A] [answered]

**Comment by ato** [comment, marked as answer]
Currently there are no options to change authentication type. (The relevant code is [here](https://github.com/internetarchive/heritrix3/blob/a151937d4c9c0a9beceeb306161e66ea4b4a19f2/engine/src/main/java/org/archive/crawler/Heritrix.java#L358-L363) if someone would like to implement one.)
